### ISSUE TITLE
json中带空格 JSONValidator的validate 结果不准

### ISSUE THREAD

**Issue by yuvenhol** [alibaba/fastjson#3603]
ex:
json ={    "5": [        "award",        "1041067"    ]};
JSONValidator.from(json).validate()
result: false

**Comment by WanaxYoung** [comment]
可以这么做：
JSONValidator.from(json).setSupportMultiValue(true).validate()

**Comment by ClickListener** [comment]
@yuvenhol 这个问题你解决了么
@WanaxYoung 我通过你这个方法，并没有解决。我断点发现问题出现在这里：
```
final void skipWhiteSpace() {
            if (ch > '\r') {
                return;
            }

            while (isWhiteSpace(ch)) {
                next();
            }
        }
```
`UTF16Validator`重写了这个方法， 当遇到' '是大于 '\r'的， 我不太明白为什么会有这个判断。

**Comment by Certseeds** [comment]
@yuvenhol @WanaxYoung @ClickListener 
Is there anyone had done something in this issue?
My group are willing to solve it in a few months.

**Comment by Certseeds** [comment]
@Linyun0307

**Comment by ClickListener** [comment]
@Certseeds NO, I haven't solve this problem, I just let json string from server strictness

**Comment by Certseeds** [comment]
@ClickListener 
这个应该和 #3671 是一个原因，已经解决了.
